django/db/models/sql/query.py

```diff
@@ -201,6 +201,7 @@ def __init__(self, model, where=WhereNode, alias_cols=True):
 
         # SQL annotation-related attributes
         self.annotations = {}  # Maps alias -> Annotation Expression
+        self.overriden_summary_annotations = {}  # Maps alias -> Overriden Annotation Expression
```

Comment:
hey @David-Wobrock thanks for the patch, just pointing out the spelling is "overridden". Would be good to update throughout.

Reply:
Oops, my bad. Should be fixed :)